radio/src/opentx.cpp

```diff
@@ -492,23 +483,20 @@ void modelDefault(uint8_t id)
 #if defined(FRSKY_RELEASE)
   g_model.moduleData[INTERNAL_MODULE].type = IS_PXX2_INTERNAL_ENABLED() ? MODULE_TYPE_ISRM_PXX2 : MODULE_TYPE_XJT_PXX1;
   g_model.moduleData[INTERNAL_MODULE].channelsCount = defaultModuleChannels_M8(INTERNAL_MODULE);
+  #if defined(EEPROM)
+    g_model.header.modelId[INTERNAL_MODULE] = findNextUnusedModelId(id, INTERNAL_MODULE);
+    modelHeaders[id].modelId[INTERNAL_MODULE] = g_model.header.modelId[INTERNAL_MODULE]
+  #endif
 #endif
 
 #if defined(PCBXLITE)
   g_model.trainerData.mode = TRAINER_MODE_MASTER_BLUETOOTH;
 #endif
 
-#if defined(EEPROM)
-  for (int i=0; i<NUM_MODULES; i++) {
-    modelHeaders[id].modelId[i] = g_model.header.modelId[i] = id+1;
```

Comment:
no more initialization of the modelID needed, as the module is now OFF by default